fppanalysis/time_delay_estimation.py

```diff
@@ -0,0 +1,153 @@
+from scipy.stats import gamma
+import correlation_function as cf
+
+
+def estimate_delays(x, y, dt, distribution=gamma, ax=None, plot="full", min_cutoff=0):
+    """
+    Use:
+       estimate_delays(x, y, dt, distribution=gamma, ax=None, plot="full", min_cutoff=0)
+
+    Estimates the time delay distribution parameters for the pulses propagating from two measurement points.
+    This is done by optimizing the time delay distribution parameters such that the predicted cross-correlation,
+    given by the convolution of the autocorrelation and the time delay distribution, best fits the observed cross-
+    correlation.
+
+    Time series x, y, corresponding to each point measurement must be provided. The time delay distribution
+    is assumed to follow a family distribution given by the argument distribution. Plots relevant autocorrelation
+    and cross-correlation functions if a matplotlib ax is provided.
+
+    Input:
+        x: Time series ...................... (N,) np array
+        x: Time series ...................... (N,) np array
+        distribution: Assumed distribution .... class implementing scipy.stats.rv_continuous
+        ax: Optional, if a matplotlib.pyplot.axis is provided, relevant plots will be plotted. These are meant to help
+        understand the underlying principles of the optimization, the plots are not suitable for scientific publication
+        .
+        plot: String, if "full", plots more stuff.
```

Comment:
Maybe say a few words what is plotted in addition. Couldn't that be a boolean since there is no other option than "full"?

Reply:
Oh, yeah I forgot to update this part!